fuse/dl/README.md

```diff
@@ -0,0 +1,75 @@
+# FuseMedML DL package
+
+The fuse.dl package contains core DL related modules that make heavy use of existing DL libraries. Currently, PyTorch and PyTorch Lightning are the two primary frameworks that FuseMedML is based on.
```

Comment:
Emphasize that it is PyTorch based DL
Maybe we should mention its role.
Something like (I'm assuming that you will articulate it better, 
* facilitate PyTorch-based models and training process
* It uses FuseMedML main concept and provides generic components that can be used to build the dl pipeline.
* It works great together with the data package and evaluation package.